packages/react/src/create-strict-api/__tests__/index.test.tsx

```diff
@@ -152,6 +181,37 @@ describe('createStrictAPI()', () => {
 
       expect(getByTestId('div')).toHaveCompiledCss('val', 'ok', { target: ':hover' });
     });
+
+    it('should type error when using values not the styles scope', () => {
+      const styles = cssMap({
+        primary: {
+          color: 'var(--ds-text)',
+          accentColor: 'red',
+          // NOTE: This should error, but doesn't until the other errors are resolved, only one error at a time
+          asdf: 'red',
+          '&:hover': {
+            color: 'var(--ds-text-hover)',
+            // @ts-expect-error — Type 'string' is not assignable to type 'undefined'.ts(2322)
+            asdf: 'red',
+          },
+          // NOTE: This should error, but desn't until the other errors are resolved, only one error at a time
```

Comment:
Can you extract this test another 3 times so we can assert it errors as expected in each location?